modules/apps/callbacks/ibc_middleware_test.go

```diff
@@ -774,7 +774,7 @@ func (s *CallbacksTestSuite) TestProcessCallback() {
 			func() {
 				executionGas := callbackData.ExecutionGasLimit
 				expGasConsumed = executionGas
-				callbackExecutor = func(cachedCtx sdk.Context) error {
+				callbackExecutor = func(cachedCtx sdk.Context) error { //nolint:unparam
```

Comment:
how are these considered unused params?

Reply:
its crying because we return nil here so in a sense complaining that returning err isn't needed. Don't ask me to rationalize for a linter tho